src/Components/Archived-card/ArchivedCard.jsx

```diff
@@ -0,0 +1,26 @@
+import "../../css/main.css";
+import "../Display-card/displaycard.css";
```

Comment:
need consistency in directory and file name, eg: dir: DisplayCard   file: DisplayCard,  right now you are using hyphen for directory name and lower case

Reply:
Done